.github/workflows/workspace_snippet.sh

```diff
@@ -5,8 +5,8 @@ set -o errexit -o nounset -o pipefail
 # Set by GH actions, see
```

Comment:
How about merge this file into the `release.yaml` totally? Now it only handle the WORKSPACE template substitution, aka, one thing but not reused. And the escapes in it is really ugly here. I think it should be extracted to file `release_notes_tpl.txt`
```markdown
## `WORKSPACE` code
```starlark
load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
http_archive(
    name = "rules_cuda",
    sha256 = "${SHA}",
    strip_prefix = "rules_cuda-${TAG}",
    urls = ["https://github.com/bazel-contrib/rules_cuda/releases/download/${TAG}/rules_cuda-${TAG}.tar.gz"],
)
```

and use `sed` or `envsubst` to expand it

Reply:
It does still pull in components of the examples workspace, too. I can go either way... do we benefit from this flexibility for any additional changes? Many of the other `rules_XYZ` projects follow this pattern (though admittedly, some, like `rules_rust` follow the pattern you're recommending). If you think this is the right time to make this change, I'll do it, otherwise I have a slight preference for minimizing the scope of the change.

Reply:
Lets follow `rules_rust`

Reply:
Done.